app/main.py

```diff
@@ -1,25 +1,69 @@
 from abc import ABC
+from typing import Type
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+        self.value: int = 0
+
+    def __get__(self, instance: object, owner: Type) -> int:
+        return self.value
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError("Value must be an integer.")
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError(
+                f"Value must be between {self.min_amount} "
+                f"and {self.max_amount}."
+            )
+        self.value = value
+
+    def __set_name__(self, owner: Type, name: str) -> None:
+        self.name = name
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
```

Comment:
The `Visitor` class assigns values directly to `age`, `weight`, and `height` without using the `IntegerRange` descriptors. Consider using the descriptors to ensure that the values are validated upon assignment.